.github/workflows/create-release-build.yaml

```diff
@@ -0,0 +1,72 @@
+# This starter workflow is for a CMake project running on a single platform.
+name: Create Release Build
+
+on:
+  push:
+    # Sequence of patterns matched against refs/tags
+    tags:
+      - 'v*' # Push events to matching v*, i.e. v1.0, v20.15.10
```

Comment:
Will this match any string containing a 'v'? Do you need a start of string anchor like "^v*"?

Reply:
Good question. Actually, this filter [accept glob patterns](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet), not regular expressions.  And glob patterns match start from beginning. For example, "v*" matches "vabc", but does not "abcvabc", to match abcvabc, we need "*v*".

Here is an [example](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-including-branches-and-tags) to match a tag whose name **starts** with "v1" (the last bullet).